composer.json

```diff
@@ -40,9 +40,9 @@
         "pavlakis/seaudi": "~0.1",
         "paragonie/csp-builder": "^2.0",
         "pavlakis/csp-middleware": "^0.1.0",
-        "shaunhare/meetup-cache":"^1",
+        "shaunhare/meetup-cache":"^1.0.4",
+        "ext-pdo": "*",
```

Comment:
How come we now need this? Is it not provided as part of any other dependencies?

Reply:
Hopefully this will shed some light: https://medium.com/@luis.barros.nobrega/ext-json-is-missing-in-composer-json-what-is-this-700a1fcad966

Reply:
thought we had bumped the cache before - must have been on a different branch